OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -1016,20 +1016,27 @@ void TracerThread::ProcessDeferredEvents() {
   pthread_setname_np(pthread_self(), "Proc.Def.Events");
   bool should_exit = false;
   while (!should_exit) {
+    ORBIT_SCOPE("ProcessDeferredEvents iteration");
```

Comment:
For consistency with above, name this TracerThread::ProcessDeferredEvents?

Reply:
`TracerThread::ProcessDeferredEvents` is too long in the ui and `ProcessDeferredEvents` is quite a unique name, while `Run` isn't (but I wanted to change `Tracer` as it was too generic).

Reply:
I had actually called this "TracerThread::ProcessDeferredEvents" and then decided against it for the reason I mentioned.

Reply:
I generally think it's better to have a clear consistent rule for naming stuff like this, but I get the reasoning.

Reply:
I'm generally all for consistency but as I said I tried the longer version and it's really a bit too long for the use we do of this, so I really prefer the current one.